src/main/java/com/entropyteam/entropay/employees/services/AssignmentService.java

```diff
@@ -1,23 +1,23 @@
 package com.entropyteam.entropay.employees.services;
 
+import java.time.LocalDate;
+import java.util.Optional;
 import java.util.UUID;
+
+import com.entropyteam.entropay.employees.models.*;
```

Comment:
Avoid this kind of import (models.*)
Please import each model explicitly:
```
import com.entropyteam.entropay.employees.models.Assignment;
import com.entropyteam.entropay.employees.models.Employee;
import com.entropyteam.entropay.employees.models.Project;
import com.entropyteam.entropay.employees.models.Role;
import com.entropyteam.entropay.employees.models.Seniority;
```

Reply:
it's fixed